tests/clients/layer0_test_client.go

```diff
@@ -1,18 +1,20 @@
 package clients
 
 import (
-	"testing"
-
 	"github.com/quintilesims/layer0/cli/client"
 	"github.com/quintilesims/layer0/common/models"
 )
 
+type Tester interface {
+	Fatal(...interface{})
+}
+
 type Layer0TestClient struct {
-	T      *testing.T
+	T      Tester
```

Comment:
I remember making this sort of change in the past - I don't now remember why it didn't stick. xD

Good change, though. :+1: